package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "inverted-index",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/andela-oaromokeye/checkpoint-inverted-index.svg?branch=master)](https://travis-ci.org/andela-oaromokeye/checkpoint-inverted-index)",
+  "main": "inverted-index.js",
+  "devDependencies": {
+    "eslint": "^3.14.1",
+    "eslint-config-airbnb-base": "^11.0.1",
+    "jshint": "^2.9.4"
+  },
+  "dependencies": {
+    "browser-sync": "^2.18.6",
+    "browserify": "^14.0.0",
+    "browserify-shim": "^3.8.13",
+    "connect": "^3.5.0",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.14.1",
```

Comment:
**Nitpick**
eslint appears in both dependencies and devdependencies
I think these should be in dev dependencies too
+ "eslint-config-airbnb": "^14.0.0",
 +    "eslint-plugin-import": "^2.2.0",
 +    "eslint-plugin-jsx-a11y": "^3.0.2",
 +    "eslint-plugin-react": "^6.9.0",`
there might be others.

http://zhiye.li/2014-06-23-npm-dependencies-and-devDependencies.html
http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies